server/base/src/main/java/org/apache/accumulo/server/conf/TableConfiguration.java

```diff
@@ -178,6 +178,14 @@ private static CompactionDispatcher createCompactionDispatcher(AccumuloConfigura
     CompactionDispatcher newDispatcher = Property.createTableInstanceFromPropertyName(conf,
         Property.TABLE_COMPACTION_DISPATCHER, CompactionDispatcher.class, null);
 
+    if (newDispatcher == null) {
+      // return early to prevent NPE
+      log.error(
```

Comment:
Not sure if this matters.  The log level of error here does not match the log level of the message in `ConfigurationTypeHelper.getClassInstance()` which is warn.  The log message from `ConfigurationTypeHelper.getClassInstance()` will probably contain the information the user needs to determine why the compaction dispatcher plugin did not load.

Reply:
Before this change the log file was getting peppered with NPE stack traces from below. This change was to make a single line log entry. I can't tell if you are making a suggestion with your comment on changing the log level or removing this, or just stating that the user should be able to find the information in the log as suggested in this log entry.